p5/visualTests/sanityTests/2DSanityTests/test23_skia.py

```diff
@@ -0,0 +1,34 @@
+from p5 import *
+
+mx = 0
+my = 0
+easing = 0.05
+radius = 24
+edge = 100
+inner = edge + radius
+
+def setup():
+        size(640, 360)
+        no_stroke()
+        ellipse_mode("RADIUS")
+        rect_mode("CORNER")
+
+def draw():
+        background(51)
+
+        global mx, my, easing, radius, edge, inner
+
+        if abs(mouse_x - mx) > 0.1:
+                mx = mx + (mouse_x - mx) * easing
+        if abs(mouse_y - my) > 0.1:
+                my = my + (mouse_y- my) * easing
+
+        mx = constrain(mx, inner, width - inner)
+        my = constrain(my, inner, height - inner)
+        fill(76)
+        rect(edge, edge, width - 2*edge, height - 2*edge)
+        fill(255)
+        ellipse([mx, my], radius, radius)
+
+if __name__ == '__main__':
+        run(renderer='skia')
```

Comment:
Yes, we should do that. Earlier I considered calling both the backend renderers one by one, something like
```
if __name__ == '__main__':
    run(renderer='skia')    
    run()
```
But, this was breaking in tests with `noLoop` enabled, because we are running the test for `100` frames and then calling  `exit()`.

I think it would be easier your way. I will make the updates to the testing suite.

Reply:
I have updated the testing suite, and the GHA for sanity tests will fail for the skia renderer now as we have not implemented some APIs in skia in this PR. We could either not run tests for skia.

Reply:
Yeah, I think not running tests for skia is a reasonable approach for now.